components/TProfile.vue

```diff
@@ -117,15 +125,25 @@
             {{ profile.name }}
           </div>
           <div class="ml-2 text-sm text-gray-700">
-            <span v-if="profile.height">• {{ profile.height }}cm</span>
-            <span v-if="profile.weight">• {{ profile.weight }}kg</span>
+            <span v-if="profile.height"
+              >•
+              {{
+                $t('myprofile.profile.height', { height: profile.height })
+              }}</span
+            >
+            <span v-if="profile.weight"
+              >•
+              {{
+                $t('myprofile.profile.weight', { weight: profile.weight })
+              }}</span
+            >
           </div>
         </div>
         <TButton
           v-if="uid !== profile.id"
           type="primary"
           :to="`/chat/${profile.username}`"
-          >Chat</TButton
+          >{{ $t('chat.title') }}</TButton
```

Comment:
That's a button, means it's a call-to-action, means it's a verb. It can be also translated as "Write a message" for example. `chat.title` is a name of the Chat page, which is a noun. That should be changed.